src/resources/videos.md

```diff
@@ -157,11 +157,13 @@ To include your course, [submit a PR][]:
 * [The Complete 2021 Flutter Development Bootcamp Using Dart][] by App Brewery
 * [Flutter & Dart - the Complete Guide, 2021 edition][]
 * [Flutter Crash Course][] by Nick Manning
-* [Flutter leicht gemacht 2022 - Zero to Mastery!][] by Max Berktold
+* [Flutter leicht gemacht 2022 - Zero to Mastery!][] by Max Berktold (German)
+* [Flutter Zero to Hero][] by Veli Bacik (Turkish)
 
 [Flutter & Dart - the Complete Guide, 2021 edition]: https://www.udemy.com/course/learn-flutter-dart-to-build-ios-android-apps/
 [The Complete 2021 Flutter Development Bootcamp Using Dart]: https://www.appbrewery.co/p/flutter-development-bootcamp-with-dart/
 [Flutter Crash Course]: https://fluttercrashcourse.com/
 [Flutter leicht gemacht 2022 - Zero to Mastery!]: https://www.udemy.com/course/dart-flutter-leicht-gemacht/
+[Flutter Zero to Hero]: {{site.youtube.com}}/watch?v=lpvuM9lo3HU&list=PL1k5oWAuBhgXdw1BbxVGxxWRmkGB1C11l
```

Comment:
Given the link checker errors, I'm guessing this host variable is wrong.

Reply:
@domesticmouse yep i've checked this document and i think this link could be this : {{site.youtube-site}}. i'm going to update this.

Reply:
fixed @domesticmouse thank u